docs/extend/update-2_0.md

```diff
@@ -139,6 +165,11 @@ The search system has been refactored to allow for more flexibility and extensib
 * You can now get the total search result count from `SearchResults`.
 * You can now replace an existing filter implementation.
 
+### Miscellaneous
+
+##### <span class="breaking">Breaking</span>
+* The `(Extend\Notification)->type()` extender no longer accepts a serializer as second argument.
```

Comment:
nothing, before it was 3 args, now it's 2 args (without the serializer arg)

The new implementation can infer the API resource from the notification blueprint, since both blueprint and resource classes specify the model class.